### PR TITLE
feat: [CHAOS-702]: Exported duration component to Chaos Child MFE

### PR DESCRIPTION
**Pull request by SahilKr24**
Signed-off-by: Sahil <sahil@harness.io>

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by SahilKr24** [comment]
retrigger cypress-pipeline

**Comment by SahilKr24** [comment]
retrigger jest

**Review by arkajyotiMukherjee — DISMISSED** [review]

**Review by satyam-harness — APPROVED** [review]

**Comment by SahilKr24** [comment]
retrigger cypress-cv

**Review by S-ayanide — APPROVED** [review]

File: src/modules/75-chaos/RouteDestinations.tsx [hunk 1]
```diff
@@ -77,6 +77,7 @@ import { MinimalLayout } from '@common/layouts'
 import { LicenseRedirectProps, LICENSE_STATE_NAMES } from 'framework/LicenseStore/LicenseStoreContext'
 import { ModuleName } from 'framework/types/ModuleName'
 import { RedirectToSubscriptionsFactory } from '@common/Redirects'
+import { Duration } from '@common/exports'
```
Comment (abhinavrastogi-harness) [inline src/modules/75-chaos/RouteDestinations.tsx:80]: why import from `exports`? why not directly from the Duration component?
direct imports will keep the dependency tree cleaner.

**Review by abhinavrastogi-harness — APPROVED** [review]

File: src/modules/75-chaos/RouteDestinations.tsx (hunk 1 above)
Comment (SahilKr24) [reply src/modules/75-chaos/RouteDestinations.tsx:80]: It also imported from that. No specific reason. Will update in next PR.

**Pull request merged**